terminus

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
+<?php
+
+/**
+ * If we're running from phar load the phar autoload file.
+ */
+$pharPath = \Phar::running(true);
+if ($pharPath) {
+    require_once "$pharPath/vendor/autoload.php";
+} else {
+    if (file_exists(__DIR__.'/vendor/autoload.php')) {
+        require_once __DIR__.'/vendor/autoload.php';
+    } elseif (file_exists(__DIR__.'/../../autoload.php')) {
+        require_once __DIR__ . '/../../autoload.php';
+    }
+}
+
+$discovery = new \Consolidation\AnnotatedCommand\CommandFileDiscovery();
```

Comment:
It is not necessary to use a CommandFileDiscovery object; you can also hardcode a list of classes containing commands, and pass that to the runner.  Using the discovery class is convenient for built-in commands, and necessary if plug-in commands are supported, though.